pkg/storaged/dialog.jsx

```diff
@@ -506,6 +506,12 @@ export const dialog_open = (def) => {
             update();
         },
 
+        get_options: (tag) => {
+            for (const f of fields)
+                if (f.tag == tag)
+                    return f.options;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230531-081900-df2e1047-fedora-testing-daily//Coverage/lcov/github-pr.diff.gcov.html)